src/cookbooks/active_dau.md

```diff
@@ -1,14 +1,14 @@
 # Active DAU and Active MAU
 
 An **Active User** is defined as a client who has `total_daily_uri` >= 5 URI for a given date.
-* Dates are defined by `submission_date_s3`.
+* Dates are defined by `submission_date_s3` or `submission_date`.
 * A client's `total_daily_uri` is defined as their sum of `scalar_parent_browser_engagement_total_uri_count` for a given date<sup>[1](#total_uri_count)</sup>.
 
-**Active DAU** is the number of Active Users on a given day.
+**Active DAU** (aDAU) is the number of Active Users on a given day.
 
-**Active MAU** is the number of unique clients who have been an Active User on any day in the last **28 days**. In other words, any client that contributes to Active DAU in the last 28 days would also contribute to Active MAU for that day. Note that this is not simply the sum of Active DAU over 28 days, since any particular client could be active on many days.
+**Active MAU** (aMAU) is the number of unique clients who have been an Active User on any day in the last **28 days**. In other words, any client that contributes to aDAU in the last 28 days would also contribute to aMAU for that day. Note that this is not simply the sum of aDAU over 28 days, since any particular client could be active on many days.
 
-For quick analysis, using `clients_daily_v6` is recommended. Below is an example query for getting Active DAU (aDAU) using `clients_daily_v6`.
+For quick analysis, using `clients_daily_v6` is recommended. Below is an example query for getting aDAU using `clients_daily_v6`.
```

Comment:
```suggestion
For quick analysis, it's possible to extract aDAU directly from `clients_daily_v6`, but note that calculating aMAU from the same data is significantly more complicated and expensive due to the need for a self-join over a 28 day window. Below is an example query for getting aDAU using `clients_daily_v6`.
```

Reply:
If you think including this caveat is useful, it should also appear in the DAU/MAU doc.